UnicornRender/video/include/unicorn/video/vulkan/Renderer.hpp

```diff
@@ -97,10 +97,13 @@ class Renderer : public video::Renderer
     void Deinit() override;
     bool Render() override;
     bool RecreateSwapChain();
-    Mesh* SpawnMesh(Material const& material) override;
+    void AddMesh(Mesh* mesh) override;
+    void AddMeshes(std::list<Mesh*> const& meshes) override;
     bool DeleteMesh(Mesh const* pMesh) override;
+    bool DeleteMeshes(std::list<Mesh*> const& meshes) override;
     void SetDepthTest(bool enabled) override;
 private:
+
```

Comment:
why extra whiteline? I would prefer to see whiteline _before_ `private:` than after